_templates/component/new/wdio-spec.js.t

```diff
@@ -0,0 +1,13 @@
+---
+to: src/components/<%= name %>/test/wdio/default.js
+---
+const { expect } = require('chai');
+const { assertScreenMatches } = require('../../../../utils/webdriverutils');
+
+describe('default', () => {
+    it('should have the right title', () => {
+        browser.url('/iframe.html?id=<%= name.toLowerCase() %>--default');
+        var screenshots = browser.checkDocument();
+        assertScreenMatches(screenshots);
+    });
+});
```

Comment:
A few things here:
- use 2 spaces for indentation
- remove semicolons

Reply:
I ran the linter on this file. Should have picked it up now